ts/protocol/otf_message_handler.py

```diff
@@ -79,11 +79,11 @@ def create_predict_response(
         if context is None:
             msg += struct.pack("!i", 0)  # content_type
         else:
-            if "true" == context.get_request_header(idx, "ts_stream_next"):
-                if ts_stream_next is False:
+            if ts_stream_next is True:
+                context.set_response_header(idx, "ts_stream_next", "true")
+            else:
+                if "true" == context.get_response_headers(idx).get("ts_stream_next"):
```

Comment:
Curious: why do we need to check this? When would this be false?

Reply:
there are 2 scenarios when ts_stream_next is false.
1. non-stream response.
2. last stream response. Line 85 is to handle this case.

context.get_response_headers(idx).get("ts_stream_next") is a string, not bool. it's value can be:
"true": has next stream
"false": last stream
"": non-stream response.